{{cookiecutter.repostory_name}}/envs/prod/.vuln.env.template

```diff
@@ -0,0 +1,5 @@
+ENV=prod
```

Comment:
this is only used by the dsn, right? what's the idea here? to use the same dsn as the project? if so then the ENV should default to `vulnrelay-prod`.

Reply:
The idea was to use the same sentry project for all servers, but use the environment field to differentiate between them.
And the environment is being constructed from DD_PRODUCT and ENV:
```
    environment = f"{settings.DD_PRODUCT} - {settings.ENV}"
```

Reply:
using the same dsn for all project is a security risk. I'd rather have them separate, even if separated from the project's DSN

Reply:
I don't get it, how exactly are multiple sentry DSNs safer than a single one?
When it comes to various bugs that may arise in production, having all of them in a single place seems much more convenient to manage. Also, creating a separate sentry setup for each new project we ship can be daunting.

Reply:
the security risk is: if one DSN gets compromised a malicious actor can spawn errors for all nodes, and cycling the DSN would require changing configs of all deployments. If creating projects is troubling then it is a problem we can solve with a steady supply of evaluation candidates - sentry API allows for creation of projects. It could be a part of CADM or whatever.